tests/e2e/system/test_mcg_recovery.py

```diff
@@ -16,6 +17,7 @@
 @tier4a
 @ignore_leftovers
 @polarion_id("OCS-2716")
+@skipif_ocs_version("<4.9")
```

Comment:
Since we changed the terms from 4.9 shouldn't it be skipif_odf_version("<4.9"), So this marker can be widely used by ocs-qe

Reply:
@petr-balogh what do you think?

Reply:
We have OCS on a lot of places, we will not change this now, for sure not in this PR. Also whole project is named OCS-CI ;) we also didn't rename to ODF-CI